board/piksiv3/rootfs-overlay/etc/init.d/S83zmq_file_logger

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
I just wanted it to start up before the firmware so it could log any start up errors. Unfortunately, there's no space currently to fit it after the upgrade and before the firmware.

I can shift things around, but more importantly I don't think we have a good way to kill this during an update since it would just be restarted by monit. I should focus on getting the settings to work, then it would possible for the update process to disable it that way. It seems unlikely that the USB contention would be an issue since the USB read for the update takes about one second, but it probably doesn't hurt to figure this out.